src/Sylius/Bundle/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -49,6 +49,7 @@ public function getConfigTreeBuilder(): TreeBuilder
             ->children()
                 ->scalarNode('driver')->defaultValue(SyliusResourceBundle::DRIVER_DOCTRINE_ORM)->end()
                 ->booleanNode('prepend_doctrine_migrations')->defaultTrue()->end()
+                ->booleanNode('shipment_processor_before_prices_recalculator')->defaultFalse()->end()
```

Comment:
I'm not sure about this flag name. Do you have any better ideas?

Reply:
i am not sure if better but different :D `processing_shipments_before_prices` or `processing_shipments_before_prices_in_checkout`

Reply:
`processing_shipments_before_prices` is shorter and expresses literally the same thing so I'm going with this one, thanks :slightly_smiling_face:

Reply:
I would be for sth like:
```suggestion
                ->booleanNode('process_shipments_before_recalculating_prices')->defaultFalse()->end()
```

Reply:
Hmm, so maybe `process_shipments_before_prices`?